src/messaging/tests/echo/entrypoint.sh

```diff
@@ -10,7 +10,7 @@ ot-ctl ifconfig up
 ot-ctl thread start
 
 if [ "$1" = "responder" ]; then
-    chip-echo-responder
+    gdb -return-child-result -q -ex run -ex bt chip-echo-responder
```

Comment:
This needs `-batch` on all instantiations to quit on backtrace in CI.

Also needs `-ex "thread apply all bt"` to make sure you get all stack traces

Reply:
I would like to hold "thread apply all bt" until we find we do need those extra thread info. 

I found I got too many debug log on various system threads like bellow. And I am not able to easily to identify the backtrace we needed. 

Thread 5 (Thread 0x7ffff5716700 (LWP 1036688)):
#0  __libc_recv (flags=<optimized out>, len=4096, buf=0x7ffff5714c80, fd=7) at ../sysdeps/unix/sysv/linux/recv.c:28
#1  __libc_recv (fd=7, buf=0x7ffff5714c80, len=4096, flags=0) at ../sysdeps/unix/sysv/linux/recv.c:23
#2  0x000055555556991a in chip::DeviceLayer::PlatformManagerImpl::WiFIIPChangeListener() () at ../../src/platform/Linux/PlatformManagerImpl.cpp:77
#3  0x000055555556c02a in std::__invoke_impl<void, void (*)()>(std::__invoke_other, void (*&&)()) (__f=@0x55555560cc18: 0x55555556980a <chip::DeviceLayer::PlatformManagerImpl::WiFIIPChangeListener()>) at /usr/include/c++/9/bits/invoke.h:60
#4  0x000055555556bfc2 in std::__invoke<void (*)()>(void (*&&)()) (__fn=@0x55555560cc18: 0x55555556980a <chip::DeviceLayer::PlatformManagerImpl::WiFIIPChangeListener()>) at /usr/include/c++/9/bits/invoke.h:95
#5  0x000055555556bf54 in std::thread::_Invoker<std::tuple<void (*)()> >::_M_invoke<0ul>(std::_Index_tuple<0ul>) (this=0x55555560cc18) at /usr/include/c++/9/thread:244
#6  0x000055555556bf11 in std::thread::_Invoker<std::tuple<void (*)()> >::operator()() (this=0x55555560cc18) at /usr/include/c++/9/thread:251
#7  0x000055555556bee2 in std::thread::_State_impl<std::thread::_Invoker<std::tuple<void (*)()> > >::_M_run() (this=0x55555560cc10) at /usr/include/c++/9/thread:195
#8  0x00007ffff77d3d84 in  () at /lib/x86_64-linux-gnu/libstdc++.so.6
#9  0x00007ffff7f9b609 in start_thread (arg=<optimized out>) at pthread_create.c:477
#10 0x00007ffff762d293 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95